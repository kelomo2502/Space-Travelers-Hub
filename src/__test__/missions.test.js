import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configurestore';
import Missions from '../components/missions';

it('Renders Rockets component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Missions />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
