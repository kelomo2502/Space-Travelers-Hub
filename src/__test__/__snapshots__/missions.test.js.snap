// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Rockets component correctly 1`] = `
<div
  className="mission-cont container"
>
  <table
    className="tab-container table"
  >
    <thead>
      <tr
        className="tab-row"
      >
        <th
          className="mission-head"
        >
          Mission
        </th>
        <th
          className="desc-head"
        >
          Description
        </th>
        <th
          className="stat-head"
        >
          Status
        </th>
        <th>
            
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
